flamingpy/codes/surface_code.py

```diff
@@ -239,9 +253,16 @@ class SurfaceCode:
         ec (str): the error complex ('primal' or 'dual').
         boundaries (str): the boundary conditions. The options are:
 
-            'open': ['primal', 'dual', 'dual'] for 'primal' EC
-                    ['primal', 'dual', 'primal'] for 'dual' EC
+            'open': ['primal', 'dual', 'dual'] for 'primal' EC,
+                    ['primal', 'dual', 'primal'] for 'dual' EC.
             'periodic': 'periodic' in all three directions.
+            'toric':  ['periodic', 'periodic', 'dual'] for 'primal' EC,
+                      ['periodic', 'periodic', 'primal'] for 'dual' EC.
+
```

Comment:
This looks a bit weird in the rendered Sphinx documentation. It might be worth fixing it.

Reply:
This is how it looks now: 👀

![image](https://user-images.githubusercontent.com/675763/173932589-8b7f18a9-9413-42ca-a425-24e5b27fd312.png)